### ISSUE TITLE
Latex bug with minus and Mul

### ISSUE THREAD

**Issue by proy87** [sympy/sympy#9202]
Latex printer can't handle expressions like -(x+5).

``` python
from sympy.printing.latex import latex
from sympy.abc import x
from sympy import Mul
expr = Mul(-1,x+5, evaluate=False)
latex(expr) # outputs -x+5 instead of -(x+5)
```

I looked int code and found

``` python
        if denom is S.One:
            # use the original expression here, since fraction() may have
            # altered it when producing numer and denom
            tex += convert(expr)
```

convert returns simple addition (since minus was previously taken  and now x+5 is_Add), then minus is simply prepended.

**Comment by Sumith1896** [comment]
What answer is expected? 
Is it `-(x + 5)` or `- ( x + 5 )` or something else?

**Comment by debugger22** [comment]
See this: `latex(expr) # outputs -x+5 instead of -(x+5)`

**Comment by Sumith1896** [comment]
Before fix it's printing  `- x + 5`, I just wanted to confirm the exact output required(proper spacing)?

**Comment by debugger22** [comment]
I think it should be `- x - 5` because autosimplification will `-(x + 5)` will simplify it.

**Comment by Sumith1896** [comment]
But `evaluate` is set to `False`.

**Comment by debugger22** [comment]
Oh sorry! Then I think `- (x + 5)` should be fine.

**Comment by debugger22** [comment]
Actually I'm not sure because pretty printer gives this `-(x + 5)`

**Comment by simamura87** [comment]
Answer should be -(x+5). Printer should print expression, not evaluate and print.

**Comment by debugger22** [comment]
@simamura87 the confusion is about the space after `-`.

**Comment by simamura87** [comment]
Latex ignores implicit whitespaces, isn't it?

**Comment by tokencolour** [comment]
Why is this issue still open?

**Comment by gxyd** [comment]
Closing. Fixed by #10889.
